syntax = "proto3";

message GroupColumnsNames {
  repeated string columns_names = 1;
}

message RecordBatchGroup {
  GroupColumnsNames group_columns_names = 1;
  repeated string group_columns_values = 2;
}
